boards/arm/secureiot1702/Kconfig.board

```diff
@@ -0,0 +1,5 @@
+config BOARD_SECUREIOT1702
+	bool "Microchip SecureIoT1702"
+	depends on SOC_SERIES_CEC
+	select HAS_DTS
+	select SOC_PART_NUMBER_CEC1702
```

Comment:
SOC_PART_NUMBER_CEC1702 seems undefined

Reply:
Same here. I just change this to `SOC_CEC1702`.